docs/admin-guide/admin-configuration.md

```diff
@@ -359,6 +361,62 @@ The following table lists the configurable attributes of the `lint` extension.
 If the mandatory annotations option is configured when you push an image, linter will verify that the mandatory annotations list present in the configuration is also found in the manifest's annotations list. If any annotations are missing, the push is denied.
 
 
+<a name="mgmt_config"></a>
+
+## Configuration management
+
+The `mgmt` extension provides an API endpoint for configuration management.
+
+To configure `mgmt`, add the `mgmt` attribute under `extensions` in the configuration file, as shown in the following example:
+
+```json
```

Comment:
We are also planning to remove this, and enable the URL if both search and UI are enabled. We are also not sure we want to advertise the URL for getting the configuration.